include/boost/math/tools/polynomial.hpp

```diff
@@ -290,6 +290,10 @@ class polynomial :
           m_data.push_back(point);
    }
 
+   // move:
+   polynomial(polynomial&& p)
+      : m_data(std::move(p.m_data)) { }
+
```

Comment:
Is there a way to detect if C++11 is supported and disable this constructor if it is not?